code/api/src/modules/user/model.js

```diff
@@ -14,6 +14,24 @@ module.exports = function(sequelize, DataTypes) {
     },
     role: {
       type: DataTypes.TEXT
+    },
+    bio: {
+      type: DataTypes.TEXT
+    },
+    image: {
+      type: DataTypes.TEXT
+    },
+    street: {
+      type: DataTypes.TEXT
+    },
+    city: {
+      type: DataTypes.STRING
+    },
+    state: {
+      type: DataTypes.STRING
+    },
+    zip: {
+      type: DataTypes.STRING
```

Comment:
The names used here also match up exactly with the names we are using on FE for this data, so that's probably helpful!

Reply:
great!! By chance but great!!